### DISCUSSION TITLE
execute install script accidentally on my laptop (Linux Mint 21.3 Mate desktop)

### DISCUSSION THREAD

**Discussion by Owz81** [outdoorbits/little-backup-box#293] [Q&A]
Hi!

I execute install script accidentally on my laptop (Linux Mint 21.3 Mate desktop). You should not do that in the night... I though ssh connection was open, but it was closed, so I executed it on my laptop.

After that, every thing seems working. Only one thing didn't work any more DNS resolution. 
Also `ifconfig -a` shows no DNS server and that also didn't change after put one in manually in the network manager.
A first look on the install script I see no reason why that happened.  
Has anyone a tip for me? What the reason is for that? And how I can repair that, without reinstalling the system?
Thanks a lot. 

Thomas

**Comment by outdoorbits** [comment]
Maybe you could uninstall openresolv. All the other network strings should be done by comitup. Please have a look at roots crontab.

**Comment by Owz81** [comment]
Openresolv is not installed und crontab is nothing special.

**Comment by Owz81** [comment]
Found an easy solution, I took the timeshift snapshot from Monday. Now every thing works again!
